types/emoji-mart/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for emoji-mart 2.11
+// Type definitions for emoji-mart 3.0
 // Project: https://github.com/missive/emoji-mart
 // Definitions by: Jessica Franco <https://github.com/Jessidhia>
 //                 Nick Winans <https://github.com/Nicell>
 //                 Elvis Wolcott <https://github.com/elvis-wolcott>
```

Comment:
@elviswolcott Would you mind following up and fixing this link so you get recognized as a package owner on future PRs?

Reply:
Will do

Unrelated, Gatsby Remark VSCode is amazing